build/run-docker-tests.sh

```diff
@@ -15,7 +15,8 @@ fi
 ./build/cluster-setup.sh
 
 #Get env from Docker arguments
-export SELENIUM_CLUSTER=https://`oc get route multicloud-console -n open-cluster-management -o=jsonpath='{.spec.host}'`
+installed_namespace=`oc get subscriptions.operators.coreos.com --all-namespaces | grep advanced-cluster-management | awk '{print $1}'`
```

Comment:
here bash syntax looks incorrect to me

Reply:
I can run it in local.   Can you please point which part is incorrect?   
```
root@yhliu-vm:~# bash -x ./a.sh
++ oc get subscriptions.operators.coreos.com --all-namespaces
++ grep advanced-cluster-management
++ awk '{print $1}'
+ installed_namespace=ocm
++ oc get route multicloud-console -n ocm '-o=jsonpath={.spec.host}'
+ export SELENIUM_CLUSTER=https://multicloud-console.apps.ocp4-gcp-2.gcp.dev09.red-chesterfield.com
+ SELENIUM_CLUSTER=https://multicloud-console.apps.ocp4-gcp-2.gcp.dev09.red-chesterfield.com
```